tests/browser-sessions.test.ts

```diff
@@ -0,0 +1,90 @@
+import { describe, it, expect, beforeAll, afterAll } from "vitest";
+import { AgentRunner } from "../worker/runner";
+import { AGENT_SERVER_PORT } from "../server/constants";
+
+describe("Browser Sessions API through Proxy", () => {
+  let agentRunner: AgentRunner;
+  const testPort = AGENT_SERVER_PORT + 20;
+  const testBrowserPort = 3022; // Use different browser port to avoid conflicts
```

Comment:
do the same thing we did in `testPort` here, where we use BROWSER_PORT + some constant